tests/run/worktree/test.sh

```diff
@@ -20,5 +20,14 @@ rlJournalStart
             rlRun "tmt run -ar provision -h $method report -vvv"
             rlRun "popd"
         rlPhaseEnd
+
+        rlPhaseStartTest "Ansible ($method) - check verbose attribute"
+            rlRun "pushd data/ansible"
+            rlRun "tmt run -rddd discover provision -h $method prepare \
```

Comment:
Please, enable the `finish` step as well so that `run --remove` actually removes the workdir.

Reply:
Now I understand why -r didn't work for me actually.